src/Nethermind/Nethermind.Facade/Simulate/SimulateTxMutatorTracer.cs

```diff
@@ -0,0 +1,93 @@
+// SPDX-FileCopyrightText: 2023 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Nethermind.Abi;
+using Nethermind.Core;
+using Nethermind.Core.Crypto;
+using Nethermind.Evm;
+using Nethermind.Evm.Tracing;
+using Nethermind.Facade.Proxy.Models.Simulate;
+using Nethermind.Int256;
+using Log = Nethermind.Facade.Proxy.Models.Simulate.Log;
+
+namespace Nethermind.Facade.Simulate;
+
+internal sealed class SimulateTxMutatorTracer : TxTracer, ITxLogsMutator
+{
+    public const int ExecutionError = -32015;
+
+    private static readonly Hash256 transferSignature =
+        new AbiSignature("Transfer", AbiType.Address, AbiType.Address, AbiType.UInt256).Hash;
+
+    private static readonly Address Erc20Sender = new("0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE");
```

Comment:
This address comes from spec?

Reply:
Yes it does [see ref.](https://github.com/DarkFlorist/execution-apis/blob/eth_multicallv1/docs/ethsimulatev1-notes.md#eth-transfer-logs)

Reply:
Can you change it to all lower case like in spec? `0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee`

Reply:
updated